rocprim/include/rocprim/warp/detail/warp_segment_bounds.hpp

```diff
@@ -53,7 +53,11 @@ unsigned int last_in_warp_segment(Flag flag)
     // Make sure last item in logical warp is marked as a tail
     warp_flags |= ballot_type(1) << (WarpSize - 1U);
     // Calculate logical lane id of the last valid value in the segment
+    #ifdef __HIP_CLANG_ONLY__
```

Comment:
`__HIP_CLANG_ONLY__` is defined by `hcc`. Instead of `#ifdef` there should be `#if`, see https://github.com/ROCm-Developer-Tools/HIP/blob/roc-1.8.x/include/hip/hcc_detail/hip_runtime.h#L35.

Reply:
Thanks for catching that!